core/src/test/scala/org/apache/spark/scheduler/SparkListenerSuite.scala

```diff
@@ -142,6 +142,7 @@ class SparkListenerSuite extends SparkFunSuite with LocalSparkContext with Match
     stageInfo.submissionTime should be ('defined)
     stageInfo.completionTime should be ('defined)
     taskInfoMetrics.length should be {4}
+    sc.stop()
```

Comment:
On Linux, with addition of sc.stop():
```
SparkListenerSuite:
- basic creation and shutdown of LiveListenerBus
- bus.stop() waits for the event queue to completely drain
- basic creation of StageInfo
- basic creation of StageInfo with shuffle
- StageInfo with fewer tasks than partitions
- local metrics
- onTaskGettingResult() called when result fetched remotely
- onTaskGettingResult() not called when result sent directly
- onTaskEnd() should be called for all started tasks, even after job has been killed
- SparkListener moves on if a listener throws an exception
- registering listeners via spark.extraListeners
Run completed in 53 seconds, 688 milliseconds.
Total number of tests run: 11
Suites: completed 2, aborted 0
Tests: succeeded 11, failed 0, canceled 0, ignored 0, pending 0
All tests passed.
```
Without sc.stop(), the test took a bit longer:
```
Run completed in 57 seconds, 357 milliseconds.
```